packages/manager/apps/pci-ai-endpoints/src/pages/token.page.tsx

```diff
@@ -61,9 +60,7 @@ export default function TokenPage() {
   const { t } = useTranslation('token');
   const { projectId } = useParams();
   const { pagination, setPagination } = useDataGrid();
-  const { isAdmin } = useUserInfos();
-
-  const { data: availability } = useFeatureAvailability([CREATE_TOKEN]);
+  const { isAuthorized, isLoading: isAuthLoading } = useUserInfos();
```

Comment:
If you apply my recommendation. Not decompose in hook for performance reason

```suggestion
  const { data: isAuthorized, isLoading: isAuthLoading } = useUserInfos();
```